test/pom.xml

```diff
@@ -53,12 +53,6 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
```

Comment:
Same. Please confirm if it is really needed.

Reply:
isn't necessary

Reply:
new commit done